index.php

```diff
@@ -7,10 +7,11 @@
 
 <meta property="fb:appid" content="<?php print FACEBOOK_APP_ID; ?>"/>
 <meta property="og:type" content="website" />
-<meta property="og:image" content="https://4and.me/images/header1.jpg"/>
-<meta property="og:title" content="Village X"/>
+<meta property="og:image" content="temp/IMG_1707.jpg"/>
+<meta property="og:title" content="This website distrupts extreme poverty in Africa."/>
 <meta property="og:url" content="https://4and.me/"/>
-<meta property="og:description" content="Fund Projects That Villages Choose - because everyone deserves a voice in development"/>
+<meta property="og:description" content="Village X Org is like GoFundMe for rural Africa. Villages choose and partially finance the projects."/>
+
```

Comment:
Since these are now in header.inc, we should delete them from here.

Reply:
cool. want me to do it?